src/platform/imx8ulp/include/platform/lib/dma.h

```diff
@@ -0,0 +1,31 @@
+/* SPDX-License-Identifier: BSD-3-Clause
+ *
+ * Copyright 2021 NXP
+ *
+ * Author: Peng Zhang
+ */
+
+#ifdef __SOF_LIB_DMA_H__
+
+#ifndef __PLATFORM_LIB_DMA_H__
+#define __PLATFORM_LIB_DMA_H__
+
+#define PLATFORM_NUM_DMACS	3
+
+/* max number of supported DMA channels */
+#define PLATFORM_MAX_DMA_CHAN	32
+
+#define DMA_ID_EDMA2	0
+#define DMA_ID_HOST	1
```

Comment:
PLATFORM_NUM_DMACS is 3 but we have only two IDs, this doesn't look right. Also ULP has EDMA?